react/components/TopMenu.js

```diff
@@ -103,9 +108,11 @@ class TopMenu extends Component {
     }
 
     const contentElement = this.content.current
-    if (contentElement) {
+    const extraHeadersElement = this.extraHeaders.current
+    if (contentElement && extraHeadersElement) {
       const offset = currentHeightReduction * 0.5
       contentElement.style.transform = `translate3d(0, ${offset}px, 0)`
+      extraHeadersElement.style.transform = `translate3d(0, ${offset}px, 0)`
```

Comment:
In order to make the support to older browsers you could apply this transform using the webkit prefix too. So you could add the following line
```javascript
extraHeadersElement.style.WebkitTransform = `translate3d(0, ${offset}px, 0)`
```